### PR TITLE
SCRIPTS: Fixed Claws of the Griffon Quest

### PR DESCRIPTION
**Pull request by Zenny89**
Complete fix for issue #1975

### CODE REVIEW

File: scripts/zones/Jugner_Forest_[S]/Zone.lua [hunk 1]
```diff
@@ -16,45 +16,49 @@ require("scripts/zones/Jugner_Forest_[S]/TextIDs");
...
+-----------------------------------     
+
+function onZoneIn(player,prevZone)      
+    local cs = -1;  
+    if ((player:getXPos() == 0) and (player:getYPos() == 0) and (player:getZPos() == 0)) then   
+        player:setPos(621.865,-6.665,300.264,149);
+    end 
+    
+    if (player:getQuestStatus(CRYSTAL_WAR,CLAWS_OF_THE_GRIFFON) == QUEST_ACCEPTED and player:getVar("ClawsOfGriffonProg") == 0) then
+        return 200;
```
Comment (TeoTwawki) [inline scripts/zones/Jugner_Forest_[S]/Zone.lua]: wrong method. set the `cs` var to the csid, at the end return the var.

Comment (TeoTwawki) [inline scripts/zones/Jugner_Forest_[S]/Zone.lua]: p.s. I realize that wasn't your edit here, but fix for us k?

**Comment by Zenny89** [comment]
Updated

**Comment by TeoTwawki** [comment]
Thanks @Zenny89 :smiley:

**Pull request merged**